completion_aggregator/management/commands/migrate_progress.py

```diff
@@ -0,0 +1,140 @@
+"""
+Migrate CourseModuleCompletion objects to BlockCompletion table.
+"""
+
+import logging
+import sys
+
+import celery
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey, UsageKey
+
+from django.core.management.base import BaseCommand
+from django.db.models.signals import post_save
+
+from completion.models import BlockCompletion
+from completion_aggregator import tasks
+from completion_aggregator.signals import completion_updated_handler
+
+try:
+    from progress.models import CourseModuleCompletion
+    PROGRESS_IMPORTED = True
+except ImportError:
+    PROGRESS_IMPORTED = False
+
+
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Migrate CourseModuleCompletion objects to BlockCompletion table.
+    """
+    def add_arguments(self, parser):
+        """
+        Entry point for subclassed commands to add custom arguments.
+        """
+        parser.add_argument(
+            '--from_settings',
```

Comment:
is it a typo the underscore instead of a normal dash (-)?

Reply:
Yeah.  I'll make it consistent.

Reply:
Yes.  I'll fix it.